src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -47,12 +50,23 @@ def _run_checkm2(
           )
 
     tool_file_name, genome_id_col = 'quality_report.tsv', 'Name'
-    process_genome_attri_result(output_dir,
-                                SELECTED_CHECKM2_FEATURES,
-                                genome_id_col,
-                                ids_to_files,
-                                [tool_file_name],
-                                )
+    genome_attri_docs = process_genome_attri_result(output_dir,
+                                                    SELECTED_CHECKM2_FEATURES,
+                                                    genome_id_col,
+                                                    ids_to_files,
+                                                    [tool_file_name],
+                                                    )
+
+    # create fatal error file if any genome is missing from the result file
+    fatal_tuples = []
+    error_message = f"Result missing in the CheckM2 output file."
+    success_ids = [genome_attri_doc[FLD_KBASE_ID] for genome_attri_doc in genome_attri_docs]
```

Comment:
have we actually had checkm fail yet?

Reply:
Encountered only once (with a single missing genome out of 550), that's how I realized the absence of a fatal error file for CheckM. However, upon rerunning, the result file has been successfully recovered. So technically, no failure for checkm.

Reply:
How did you test this code worked as expected in that case?

Reply:
To test it, I manually remove a random id from `success_ids` and build a customized image in my personal registry.

Reply:
:+1: